### ISSUE TITLE
Timed comment text area not resizing with popover

### ISSUE THREAD

**Issue by YossarianXXII** [sghpjuikit/player#11]
Text area of timed comment (chapter) resizes only height when resizing its parent PopOver

It is resizing based on text length like `double w = 110 + len/3;` where 110 is some artificial number.

  
![timedcommentresize](https://user-images.githubusercontent.com/5514221/34769132-44acde9e-f5fd-11e7-9c9e-c1a024349f70.png)

**Comment by YossarianXXII** [comment]
Trying to fix this but don't understand the `maintain` function - if I add a simple sout( popover.width ) inside , the app crashes when editing comment and I try to resize popover

**Comment by sghpjuikit** [comment]
The formula is for computing the area size needed to display the text in a nice looking box. It needs some work as it should take font size into consideration. The number 110 has come out as a nice 'start size' value.

There appears to be a critical bug hanging & crashing JVM without a trace, somewhere deep in JavaFX. I have come up against this elusive crash as well and it happens sometimes (only) when the chapter popover content is changed. 

As for the resize not adhering popover size during resize, I'll look into it.

**Comment by YossarianXXII** [comment]
only thing needed is for the text area

`HBox.setHgrow(ta, Priority.ALWAYS)`

tried it and.. beside that whenever I click on the edit comment button, app freezes - with debug, it went past the crash and resizing works as I believe it should. 

The freeze is the real issue here

**Comment by sghpjuikit** [comment]
I am not sure which exact method does the grow priority need to be set. Can you make the appropriate change, since you have already tested the code? Pull request is hardly necessary for this.

Are you sure the freeze is not simply because the breakpoint in debug suspends all threads (including mouse polling), which freezes the mouse as well? Make sure you set the breakpoint to suspend only the active thread. Normally this then suspends only UI thread and mouse polling and other functions work as usual (unless they interact with the ui thread, in which case they get blocked too).

**Comment by sghpjuikit** [comment]
i removed logic from root's layoutChildren() in the skin. The crashes should no longer be happening. I have been running on this version for a week or so and the fix does not appear to cause any regressions or other issues.

@YossarianXXII Please can you make a pull request that fixes the original problem?

**Comment by sghpjuikit** [comment]
Fixed

**Comment by sghpjuikit** [comment]
Regarding the freeze bug. Eventually it came back. The way to reproduce it is to have both `content` pane in PopOverSkin and the `contentNode` passed in the PopOver have `preferredSize` set. I think this triggers the problem. I have also recorded very unusual garbage collector activity (keeps gcing something forever, which causes the freeze).

I have improved handling of `updatePath()` in PopOver and the issue went away. I believe.
